libs/partners/openai/langchain_openai/chat_models/base.py

```diff
@@ -435,7 +435,7 @@ class BaseChatOpenAI(BaseChatModel):
     """Number of chat completions to generate for each prompt."""
     top_p: Optional[float] = None
     """Total probability mass of tokens to consider at each step."""
-    max_tokens: Optional[int] = None
+    max_tokens: Optional[int] = Field(default=None, alias="max_completion_tokens")
```

Comment:
do we need to update azure openai as well? if someone is passing in max_completion_tokens to azure today this change will update so instead max_tokens is passed in